rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -1091,3 +1093,29 @@ def test_add_atom_labels_for_reaction_3(self):
         self.assertIn('*2',found_labels)
         self.assertIn('*3',found_labels)
 
+
+###################################################
+class TestLoadFilterFits(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Load the database to test loading the YAML file with custom filter criteria.
+        """
+        global database
+        cls.database = database
+
+    def testLoad(self):
+        """
+        Test if loading Arrhenius fits of the highest training reaction rates in each reaction family from
+        `RMG-database/input/FilterArrheniusFits.yml` works.
+        """
+        modelSettingsList = []
+        modelSettingsList.append(ModelSettings)
```

Comment:
Thanks for redoing this test! This was exactly what I was thinking of.

Only fix is that I think you need to do `ModelSettings()` here to create an instance of the class rather than passing the class itself. You could also add a second ModelSettings instance to test that it's saved properly to multiple.
```
modelSettingsList = [ModelSettings(), ModelSettings()]
```

Reply:
Thanks, done.